src/libs/focusTextInputAfterAnimation/types.ts

```diff
@@ -0,0 +1,5 @@
+import {TextInput} from 'react-native';
+
+type FocusTextInputAfterAnimation = (inputRef: TextInput | HTMLInputElement, animationLength: number) => void;
+
+export default FocusTextInputAfterAnimation;
```

Comment:
Sorry, I am a little behind on what exactly the conventions are that we are following for TS types. Let me know if I am not current on some of these conventions.

1. The name of the default export should always match the module name (you can see this being done in all the components). Should this file really have the generic name of `types.ts`? Or should it be more like `focusTextInputAfterAnimationTypes.ts`?
2. There is a case mismatch between the module name `focus...` and the export `Focus...`

Reply:
Bump

Reply:
Hi @tgolen sorry for the delay. Here are the answers to your questions:
1. [This is what TS specification says about that case](https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#file-organization) 
2. According to eslint configuration as well as [Expensify TS style guide](https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#naming-conventions) it should be written in PascalCase

Hope this helps! 😄

Reply:
OK, thank you very much! That's helpful